### PR TITLE
Translate password tooltip and password show/hide btn

### PR DESCRIPTION
**Pull request by BryanEliDimas**
## Fixes the translation part of https://github.com/openstax/business-intel/issues/952

### CODE REVIEW

File: app/views/newflow/login_signup/_password_show_hide.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <span class="toggle-show-hide">SHOW</span>
+    <span class="toggle-show-hide" style="display: none">HIDE</span>
+</span>
+
+<script type="text/javascript">
+  $(window).ready(function() {
+      var pwd_input = document.getElementById('signup_password');
+      var pwd_checkmark = document.getElementById('password-requirements-checkmark');
+
+      pwd_input.oninput = function(event){
```
Comment (steveburkett) [inline app/views/newflow/login_signup/_password_show_hide.html.erb:11]: where is this invoked?  just not clear why it's stored here on a pwd_input as a variable.

File: app/views/newflow/login_signup/_password_show_hide.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<span id="password-show-hide-button">
+    <span class="toggle-show-hide">SHOW</span>
+    <span class="toggle-show-hide" style="display: none">HIDE</span>
+</span>
+
+<script type="text/javascript">
+  $(window).ready(function() {
+      var pwd_input = document.getElementById('signup_password');
```
Comment (steveburkett) [inline app/views/newflow/login_signup/_password_show_hide.html.erb:8]: this way, or $ way?  i see a mix of this.

Comment (BryanEliDimas) [reply app/views/newflow/login_signup/_password_show_hide.html.erb:8]: You mean this?
```
$(function() {
  // Handler for .ready() called.
});
```

File: app/views/newflow/login_signup/_password_show_hide.html.erb (hunk 1 above)
Comment (BryanEliDimas) [reply app/views/newflow/login_signup/_password_show_hide.html.erb:11]: That's the way it works? :) The function gets called `oninput`

File: app/views/newflow/login_signup/_password_show_hide.html.erb (hunk 2 above)
Comment (BryanEliDimas) [reply app/views/newflow/login_signup/_password_show_hide.html.erb:8]: Fixed. Pushed a commit.

**Pull request merged**